spring-cloud-dataflow-configuration-metadata/src/main/java/org/springframework/cloud/dataflow/configuration/metadata/container/DefaultContainerImageMetadataResolver.java

```diff
@@ -85,12 +99,24 @@ public Map<String, String> getImageLabels(String imageName) {
 					(Map<String, String>) configElement.get("Labels") : Collections.emptyMap();
 		}
 		else {
-			throw new ContainerRegistryException(
-					String.format("Image [%s] is missing manifest", imageName));
+			throw new ContainerRegistryException(String.format("Image [%s] is missing manifest", imageName));
+		}
+	}
+
+	private static int asInt(Object value) {
+		if (value instanceof Number) {
```

Comment:
Probably want to check for a null.

Reply:
`value instanceof Number` is never true when `value` is `null`

Reply:
If there is no schemaVersion or schemaVersion is 1 we want to try the other manifest type query.